llvm/test/Transforms/InstCombine/scalable-const-fp-splat.ll

```diff
@@ -13,3 +13,16 @@ define <vscale x 2 x float> @shrink_splat_scalable_extend(<vscale x 2 x float> %
   %5 = fptrunc <vscale x 2 x double> %4 to <vscale x 2 x float>
   ret <vscale x 2 x float> %5
 }
+
+define <vscale x 2 x float> @shrink_splat_scalable_extend_rhs_constexpr(<vscale x 2 x float> %a) {
+; CHECK-LABEL: define <vscale x 2 x float> @shrink_splat_scalable_extend_rhs_constexpr(
```

Comment:
Would be good to have a fixed-width variant of this too, if there isn't one already?

Reply:
Added to fpextend.ll in d26e6721df2ce5d6aa5f581c7a230ac4d4e45719